services/upload/src/test/java/app/coronawarn/server/services/federation/upload/utils/MockData.java

```diff
@@ -56,16 +56,20 @@ public static DiagnosisKey generateRandomDiagnosisKey(boolean consentToShare) {
         .withTransmissionRiskLevel(2)
         .withConsentToFederation(consentToShare)
         .withCountryCode(TEST_ORIGIN_COUNTRY)
-        .withDaysSinceOnsetOfSymptoms(1)
+        .withDaysSinceOnsetOfSymptoms(randomDaysSinceOnsetOfSymptoms())
         .withSubmissionTimestamp(12)
         .withVisitedCountries(Set.of("FR", "DK"))
         .withReportType(ReportType.CONFIRMED_TEST)
         .build();
   }
 
+  private static Integer randomDaysSinceOnsetOfSymptoms() {
+    return ThreadLocalRandom.current().nextInt(0, 13);
+  }
+
   private static byte[] randomByteData() {
     byte[] keydata = new byte[16];
-    new Random().nextBytes(keydata);
+    ThreadLocalRandom.current().nextBytes(keydata);
```

Comment:
I'd suggest to replace this with SecureRandom because Fortify will complain later about it . 🛡️